packages/opentelemetry/src/trace.ts

```diff
@@ -227,6 +225,8 @@ function getContext(scope: Scope | undefined, forceTransaction: boolean | undefi
   const { spanId, traceId } = parentSpan.spanContext();
   const sampled = getSamplingDecision(parentSpan.spanContext());
 
+  // In this case, when we are forcing a transaction, we want to treat this like continuing an incoming trace
```

Comment:
just to understand this correctly: Here we're populating a DSC at span creation time - right? Do we still update the DSC's `transaction` property when the name changes?

Reply:
We are not allowed to update the transaction name in that case because it's a transaction boundary and would break dynamic sampling. Or do you mean, when the parent transaciton's name updates?

Reply:
in this case yes, we would not update this anymore - BUT it is very likely that by this time we already have updated the span correctly, so it should be fine in 99% of cases (as this will be "frozen" at the time when the child span will be created, by which time the parent span has hopefully already "stabilized") - does this make sense?